components/org.wso2.carbon.identity.mgt.store.connector.jdbc/src/main/java/org/wso2/carbon/identity/mgt/store/connector/jdbc/connector/JDBCCredentialStoreConnector.java

```diff
@@ -260,16 +260,21 @@ public String updateCredentials(String userIdentifier, List<Callback> credential
     public void deleteCredential(String username) throws CredentialStoreConnectorException {
 
         try (UnitOfWork unitOfWork = UnitOfWork.beginTransaction(dataSource.getConnection(), false)) {
+            try {
 
-            NamedPreparedStatement deleteCredentialPreparedStatement = new NamedPreparedStatement(
-                    unitOfWork.getConnection(),
-                    sqlQueries.get(ConnectorConstants.QueryTypes.SQL_QUERY_DELETE_CREDENTIAL));
+                NamedPreparedStatement deleteCredentialPreparedStatement = new NamedPreparedStatement(
```

Comment:
Better leave the previous formatting unchanged, as it is not related to this work